src/main/java/com/capitalone/dashboard/webhook/github/GitHubCommitV3.java

```diff
@@ -221,6 +222,12 @@ public String process(JSONObject jsonObject) throws MalformedURLException, Hygie
                 commit.setFilesModified((List) cObj.get("modified"));
             }
 
+            List<RepoFile> files = (List<RepoFile>) cObj.get("files");
+            for (RepoFile file : files) {
+                String fileType = file.getFilename().split("[.]")[1];
```

Comment:
the variable is initialized inside a loop which is useless. Can you check this?

Reply:
Removing for now as Daniel is using the file type in his story.